RELEASE/scripts/autoscend/iotms/mr2024.ash

```diff
@@ -407,6 +489,24 @@ int remainingEmbers()
 	return get_property("availableSeptEmbers").to_int();
 }
 
+boolean auto_goingToMouthwashLevel()
```

Comment:
while you're here: you also won't be able to use mouthwash in Bees Hate You or G Lover

Reply:
fixed